libraries/appnexusKeywords/anKeywords.js

```diff
@@ -85,7 +97,7 @@ function convertKeywordsToANMap(kwarray) {
  * @param ortb2
  * @return {{}} appnexus-style keyword map using all keywords contained in ortb2
  */
-export function getANMapFromOrtb(ortb2) {
+export function getANMapFromOrtbKeywords(ortb2) {
```

Comment:
I think it makes sense to simplify this to get rid of the `key=value` parsing now (and treat keywords as just simple keywords); since with this there's no need for the hack anymore - but if we release it and people start using it you'll be stuck with it for a while.

Reply:
It's never been released - the only dependence on it comes from recent updates that are also still to be released with 8. But fair enough, it's better to undo this only after those are updated yet again to use segments instead.